src/index.ts

```diff
@@ -1749,6 +1749,7 @@ promisifyAll(Datastore, {
     'double',
     'isDouble',
     'geoPoint',
+    'getProjectId',
```

Comment:
It looks like this makes an actual change to the source of the library - can you use the `fix:` conventional commit prefix to make sure this triggers a launch?